src/app/app.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="app-container">
   <app-header class="app-container__header"></app-header>
-  <app-course-collection class="app-container__content"></app-course-collection>
+  <router-outlet class="app-container__content"></router-outlet>
```

Comment:
Content is placed under router-outlet so the styling is a little bit wrong because **app-container__content** class won't be applied to the real content
```suggestion
  <div class="app-container__content">
    <router-outlet ></router-outlet>
  </div>
```